yturl.py

```diff
@@ -76,9 +76,15 @@ def itags_for_video(video_id):
     try:
         streams = api_response["url_encoded_fmt_stream_map"].split(",")
     except KeyError:
-        raise NotImplementedError(
-            "Live and streaming videos are unsupported."
-        ) from None
+        try:
+            import json
+            return collections.OrderedDict([(i['itag'], i['url']) for i in
+             json.loads(api_response['player_response'])['streamingData'][
+                 'formats']])
+        except:
```

Comment:
Don't use bare except, check the exceptions you actually care about. For example, this will even catch keyboard interrupts...